experimental/GaloisGrp/GaloisGrp.jl

```diff
@@ -1717,6 +1717,24 @@ Currently the polynomial needs to be monic.
 The group is returned as an explicit permutation group permuting the roots as contained
 in the contex object (the 2nd return value). The roots live in a suitable unramifed
 extension of the p-adics.
+
+# Example
+```jldoctest
+julia> K, a = cyclotomic_field(5);
+julia> G, C = galois_group(K)
+julia> G, C = galois_group(K)
```

Comment:
Redundant line; and Julia 1.6 seems to need the empty lines in between.
```suggestion
julia> K, a = cyclotomic_field(5);

julia> G, C = galois_group(K)
```